src/main/java/com/sprints/validations/SprintsValidations.java

```diff
@@ -26,6 +26,14 @@ public void sprintValidateInBacklog(Sprint sprint){
 		}
 	}
 	
+	public void sprintValidateBothBooleans(SprintDomain sprintDomain){
```

Comment:
This validation is unnecessary , you will validate the sprint that is coming in the domain.